inspirehep/modules/authors/workflows/authornew.py

```diff
@@ -127,7 +127,7 @@ def formatter(bwo, **kwargs):
         else:
             # FIXME add a template for the author display in the HP
             return render_template("authors/workflows/authorupdate.html",
-                                   record_preview="",
+                                   record=bwo.data,
```

Comment:
I think you also need to do this change in `inspirehep/modules/authors/workflows/authorupdate.py` as well.